src/Tools/onnx-mlir-opt/onnx-mlir-opt.cpp

```diff
@@ -86,13 +75,16 @@ void scanAndSetOptLevel(int argc, char **argv) {
     int num = atoi(&argv[i][2]); // Get the number starting 2 char down.
     // Silently ignore out of bound opt levels.
     if (num >= 0 && num <= 3) {
-      OptimizationLevel = (OptLevel)num;
+      OptimizationLevel = (onnx_mlir::OptLevel)num;
       return;
     }
   }
 }
 
 int main(int argc, char **argv) {
+  // Initialize accelerators if they exist.
+  bool hasAccelerator = InitAccelerators();
```

Comment:
[nit]: hasAccelerator -> hasAccelerators. (there could be more than one in the future).

Reply:
Done. Thanks!